task-maker-format/src/ioi/dag/input_validator.rs

```diff
@@ -36,6 +36,7 @@ impl InputValidator {
         eval: &mut EvaluationData,
         description: String,
         subtask_id: SubtaskId,
+        subtask_name: &Option<String>,
```

Comment:
What about `Option<&str>`? This way you can pass `None` or `subtask.name.as_ref()`.